readme.md

```diff
@@ -1,18 +1,76 @@
-# Oanda v20 Platform - Public Version
-
-This is a fully custom forex robot algorithm library built using the python Backtrader framework as reference, with full integration into the Oanda version 20 API schema. I built this platform after dissatsifactory results with other Python/Oanda platforms I was able to find on the market.  Due to this fact, it does not natively support all Oanda API calls you may need for your use case, but it is easily scalable to suit your individual needs.
+[![Build Status](https://travis-ci.com/ant358/oanda_v20_platform_public.svg?branch=master)](https://travis-ci.com/ant358/oanda_v20_platform_public)
+[![codecov](https://codecov.io/gh/ant358/oanda_v20_platform_public/branch/master/graph/badge.svg)](https://codecov.io/gh/ant358/oanda_v20_platform_public)
+# Automated Trading Platform that runs on Oanda  
```

Comment:
This codecov is cool.  Never heard of it.  I'll learn how to set up my own account.

Reply:
Yes, I had seen it on other Github repos, but not used it myself before. The setup was easy and it plays nicely with Travis CI and pytest. It's the travis.yml that triggers the codecov analysis on a commit. For Travis you can add the .env variables as secrets (in Travis rather than Github) and the config['TOKEN'] still works. The only problem is writing more unit tests! :-)

Reply:
Super slick.  I'll play with it.  Really cool idea. smh - I'm so bad at tdd... I need to force myself to integrate more. :)